### ISSUE TITLE
Support priority definition for userTasks

### ISSUE THREAD

**Issue by Skaiir** [camunda/zeebe-bpmn-moddle#62]
### What

We should support a simple priority field as part of feature parity.

```xml
<userTask id="" name="">
  <bpmn:extensionElements>
    <zeebe:priorityDefinition priority="15" />
  </bpmn:extensionElements>
</userTask> 
```

### How

The field should be an extension element as the bpmn schema only allows integer definitions of a priority. It should also parse as a string for the same reasons. 

Related to https://github.com/camunda/product-hub/issues/217
Related to https://github.com/camunda/camunda-modeler/issues/4450

**Comment by barmac** [comment]
Released via v1.5.0

**Comment by barmac** [comment]
Reopening due to https://github.com/camunda/product-hub/issues/217#issuecomment-2275277482

I will update the top comment too.
